alphadia/outputtransform.py

```diff
@@ -678,6 +680,50 @@ def build_stat_df(
 
         return stat_df
 
+    def build_internal_df(
+        self,
+        folder_list: list[str],
```

Comment:
(nit) `search_output_folders`

Reply:
`folder list` is used consistently throughout the file. If I do change it, should I change it everywhere (and if so should this be a separate, small PR)?